tests/console/system_prepare.pm

```diff
@@ -77,6 +77,10 @@ sub run {
     # Save output info to logfile
     if (is_sle) {
         my $out = script_output("SUSEConnect --status-text", proceed_on_failure => 1);
+        if ($out =~ /System management is locked by the application with pid/) {
```

Comment:
I would suggest to limit number attempts to some timeout,e.g. we try every 5 seconds but not more than 30 seconds in total, as 2 tries might not help in some cases. 
Do you know which service actually locks sys mgmt? As we already mask services like `packagekit` not to have issue while using zypper, as it might using zypper lock at any point of time.

Reply:
> I would suggest to limit number attempts to some timeout,e.g. we try every 5 seconds but not more than 30 seconds in total, as 2 tries might not help in some cases.

Will do.

> Do you know which service actually locks sys mgmt? As we already mask services like `packagekit` not to have issue while using zypper, as it might using zypper lock at any point of time.

No idea, other than it being `/usr/bin/zypper`. Couldn't find the offending process in the logs, and also tried to reproduce manually with no luck.

Reply:
Haven't seen it fail in the past three builds either.